examples/with-docker/README.md

```diff
@@ -1,3 +1,18 @@
+# ♦️How to run KOSMO-FRONTEND:
```

Comment:
What is Kosmo-Frontend?

Reply:
Sorry it was a mistake. I was updating my turborepo readme but not sure how I ended up with this PR.

Reply:
😄 no worries!